packages/dashboard-v2/src/components/Tabs/ActiveTabIndicator.js

```diff
@@ -35,6 +35,9 @@ export const ActiveTabIndicator = ({ tabRef }) => {
   );
 };
 
-ActiveTabIndicator.propTypes = {
-  tabRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: PropTypes.instanceOf(Element) })]),
-};
+// Needed, because we're using an Element constant here which Gatsby doesn't recognize during build time.
+if (typeof window !== "undefined") {
+  ActiveTabIndicator.propTypes = {
+    tabRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: PropTypes.instanceOf(Element) })]),
+  };
+}
```

Comment:
can you use `PropTypes.elementType` instead of `Element` so we don't need to add this window condition?

Reply:
Never mind, Gatsby caching mislead me again -- after a restart it started complaining. 🙄 
There is [a feature request](https://github.com/facebook/prop-types/issues/240) in `prop-types` repo, but it never got too much traction (I guess TypeScript is more common these days).

I reverted it back to what it was, this way we get the protection & build passes (`prop-types` are runtime anyways, so we don't need them on build time, and we don't do SSR).

7f26c95b

Reply:
```
ActiveTabIndicator.propTypes = {
  tabRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: PropTypes.elementType })]),
};
```

This got me successful build. I did run gatsby clean before the build too so not sure why it failed for you.

Reply:
Build doesn't fail, but it complains when you visit the page.